app/infrastructure/queue_consumer.py

```
@@ -154,30 +244,118 @@ async def _process_job(self, worker_id: str, job_data: Dict[str, Any]):
+                repository_branch=serialized_model.get("repository_branch"),
+                job_type=serialized_model.get("job_type"),
+                job_queued_at=serialized_model.get("job_queued_at")
+            )
+            
+            email_dispatcher = get_email_dispatcher()
+            await email_dispatcher.send_templated_html(
+                to=[job_tracer.user_email],
+                template=Template.PROJECT_ANALYSIS_SUCCESS,
+                context=context,
+            )
+        except Exception:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)